media.emby.EmbyServer.json

```diff
@@ -0,0 +1,35 @@
+{
+    "app-id": "media.emby.EmbyServer",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "media.emby.EmbyServer",
+    "finish-args": ["--share=network", "--filesystem=host", "--device=dri"],
+    "modules": [
+        {
+            "name": "media.emby.EmbyServer",
+            "buildsystem": "simple",
+            "build-commands": ["tar -xf emby-server-flatpak_4.6.2.0_${FLATPAK_ARCH}.tar.xz -C ${FLATPAK_DEST}"],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://github.com/MediaBrowser/Emby.Releases/releases/download/4.6.2.0/emby-server-flatpak_4.6.2.0_aarch64.tar.xz",
+                    "sha256": "a11b91167f59f00dcbd29367f56570f92136de64343de8c130ee5189250c24b0",
+                    "only-arches": ["aarch64"]
+                },
+                {
+                    "type": "file",
+                    "url": "https://github.com/MediaBrowser/Emby.Releases/releases/download/4.6.2.0/emby-server-flatpak_4.6.2.0_arm.tar.xz",
+                    "sha256": "8ffe780e7fb88e9ee44a2bc337dbc424a4f12b8cca711c160c8e30d8894930dd",
+                    "only-arches": ["arm"]
+                },
```

Comment:
```suggestion
```

Runtime 20.08 shall be used and that's where `arm` has been dropped from supported platforms.